### DISCUSSION TITLE
Thx for the great app

### DISCUSSION THREAD

**Discussion by McDull-boot** [pilotmoon/Scroll-Reverser#102] [General]
I'm a new beginner of Mac OS , and suffer from the stupid system mouse setting(just can't figure out why can't I reserver the mouse while keeping the touch pad normal. however, thank Nick for creating a great functional tool.

**Comment by pilotmoon** [comment]
Thanks for the kind words and congrats on being the first person to try out the discussions feature :)

**Reply by McDull-boot** [reply]
Lol, well I always want to donate your work, but I can't do it through PayPal as a Chinese user :( . On the top of that, I think more function can be added to the app such as smooth scroll~

**Reply by pilotmoon** [reply]
Tell me more about that

**Reply by senhalil** [reply]
Hi Nick, is it too much hassle to activate github sponsorship (for the project or directly for you)?

**Reply by senhalil** [reply]
> Hi Nick, is it too much hassle to activate github sponsorship (for the project or directly for you)?

I "bought you a coffee" but if I could become a github sponsor I could donate via my professional account.
